playground/remix/app/providers.tsx

```diff
@@ -8,8 +8,8 @@ export function PHProvider({ children }: { children: React.ReactNode }) {
     const [events, setEvents] = useState<CaptureResult[]>([])
 
     useEffect(() => {
-        posthog.init('phc_test_key_for_playground', {
-            api_host: '/ph-relay-xyz123',
+        posthog.init('sTMFPsFhdP1Ssg', {
```

Comment:
Don't add our production key here, people already use our key on their apps just because we have it on our `posthog` repo , we don't need even more people using it 😥

Reply:
```suggestion
        posthog.init('phc_test_key_for_playground', {
```